junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileSource.java

```diff
@@ -146,4 +146,15 @@
 	@API(status = EXPERIMENTAL, since = "5.7")
 	int maxCharsPerColumn() default 4096;
 
+	/**
+	 * Identifies whether leading and trailing whitespace characters of unquoted CSV columns should be ignored.
```

Comment:
Let's clarify what a quoted CSV column is, i.e. that it starts with a quotation character.